solvation_analysis/analysis.py

```diff
@@ -0,0 +1,127 @@
+# from abc import ABC
+from abc import ABC
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.analysis.rdf import InterRDF
+from MDAnalysis.analysis import distances
+import numpy as np
+from solvation_analysis.rdf_parser import identify_solvation_cutoff
+from solvation_analysis.analysis_library import (
+    _CoordinationNumbers,
+    _IonPairing,
+    _IonSpeciation,
+)
+
+
+def some_function():
+    return
+
+
+class Solute(AnalysisBase):
+    """
+    The core class of the solvation module.
+    """
+
+    def __init__(
+        self, solute, solvents, radii=None, rdf_kernel=None, kernel_kwargs=None, **kwargs
+    ):
+        """
+        Parameters
+        ----------
+        solute (AtomGroup): an atom group  # TODO: force to atom group
```

Comment:
You're using a different docs convention from `rdf_parser` here, is this intentional?

Reply:
Not intentional 😬. Generally the documentation in this PR is a bit lacking right now. I'll reformat the existing documentation and make sure the new stuff is formatted correctly. I'll leave this unresolved until that is complete.